armi/cases/case.py

```diff
@@ -769,10 +769,16 @@ def copyInterfaceInputs(
         # guess. In future, it might be nice to have interfaces specify which
         # explicitly.
         for key, files in interfaceFileNames.items():
-            if isinstance(key, settings.Setting):
-                label = key.name
-            else:
-                label = key
+            try:
+                if not isinstance(key, settings.Setting):
+                    key = cs.getSetting(key)
```

Comment:
I think that the exception check should be `NonexistentSetting` based on the behavior of `getSetting`. Also, I'd recommend moving the try/except just around the `getSetting` call.